packages/golden-layout/src/config/ItemConfig.ts

```diff
@@ -1,27 +1,17 @@
-import type { StackHeaderConfig } from '../items/Stack';
-
-export type ItemConfigType =
-  | ItemConfig
+export type ItemConfig =
```

Comment:
Add an `export type ItemConfigType = ItemConfig` as well; otherwise this is a breaking change (since the `ItemConfigType` is disappearing).

Reply:
done